src/DynamoCore/Graph/Workspaces/SerializationConverters.cs

```diff
@@ -800,6 +800,10 @@ public override void WriteJson(JsonWriter writer, object value, JsonSerializer s
                 writer.WriteEndArray();
                 writer.WriteEndObject();
             }
+            else
+            {
+                logger.LogWarning("Unnsuccessful attempt to serialize a PackageDependencyInfo object.", Logging.WarningLevel.Moderate);
```

Comment:
Does this need to be localizable?

Reply:
Good point, we should escape the localization of it